include/mbedtls/build_info.h

```diff
@@ -134,6 +134,12 @@
 #define MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_SOME_EPHEMERAL_ENABLED
 #endif
 
+#if defined(MBEDTLS_HAVE_ASM) && defined(__GNUC__) && \
```

Comment:
Why not drop `MBEDTLS_HAVE_ASM` here? That would make the behavior consistent with the current behavior: if `AESCE_C` is enabled and the target CPU is aarch64 and `HAVE_ASM` is disabled then the build fails (because the configuration looks inconsistent).

Reply:
I actually think I'm going to go with option 1 from the description block, which means check_config won't complain on non-Arm64 but will complain on Arm64, which is I think more in line with what we want